frontend/src/pages/Interventions/Activities.tsx

```diff
@@ -0,0 +1,129 @@
+import React, {
+  FunctionComponent,
+  ReactElement,
+  useEffect,
+} from 'react';
+import Layout from '../../components/Layout';
+import { connect } from 'react-redux';
+import { getCurrentInterventions } from '../Interventions/data/reducer';
+import { Interventions } from '../../models/types';
+import { getInterventions } from './data/actions';
+import FirstActivity from '../Interventions/FirstActivity';
+import SecondActivity from '../Interventions/SecondActivity';
+import ThirdActivity from '../Interventions/ThirdActivity';
+import FourthActivity from '../Interventions/FourthActivity';
+
+interface ActivityProps {
+  interventions: Interventions;
+  getInterventions: () => void;
+}
+
+const connector = connect(
+  (state) => ({
+    interventions: getCurrentInterventions(state),
+  }),
+  {
+    getInterventions: getInterventions.request,
+  }
+);
+
+const Activities: FunctionComponent<ActivityProps> = ({
+  interventions,
+  getInterventions,
+}): ReactElement => {
+  useEffect(() => {
+    if (!interventions) getInterventions();
+  }, [interventions, getInterventions]);
+
+  const setId = interventions && interventions.setId;
+  const currentWordIdx = interventions && interventions.wordIdx;
+  const currentActivityIdx =
+    interventions && interventions.activityIdx;
+  const wordList = interventions && interventions.wordList;
+  const activities = wordList && wordList[currentWordIdx].activities;
+  const title = wordList && wordList[currentWordIdx].word.value;
```

Comment:
There should be a separate component that has the actual intervention that exists.